src/components/screens/Authenticator/Authenticator.container.vue

```diff
@@ -45,6 +45,11 @@ export default {
       type: Boolean,
```

Comment:
if i correct understand this task, you just need to remove this block of code
```
      if (SING_IN_CHALLENGE_TYPES.includes(this.challengeType)) {
        this.$emit('sign-in', { email });
        return;
      }
```
in `SignIn.interactor.vue`

and that's all

Reply:
we skip `nextScreen` call then

Reply:
current decision works and additionally provides way to detect was sign in by socials or not

Reply:
server side is not implemented.
we have a bug in the auth system?

Reply:
server side is in another tssk
current issue is about prevent `password` window from appearing when you sign in with socials

Reply:
issue-related bugs was fixed here

Reply:
server should implement first their fixes and then you need remove the code, what i've pasted.
@kazurus am i right? or this task about something other?

Reply:
@dudiq you're right. We should wait the server fix, and then only skip password forms for social + active otp

Reply:
But removing the code above doesn't solve the issue. If we have social + active otp, we should ask the code

Reply:
ok, understand